mlflow/cli.py

```diff
@@ -522,6 +550,12 @@ def gc(backend_store_uri, run_ids):
                 "Run % is not in `deleted` lifecycle stage. Only runs in "
                 "`deleted` lifecycle stage can be deleted." % run_id
             )
+        # raise MlflowException if run_id is newer than older_than parameter
+        if older_than and run_id not in deleted_run_ids_older_than:
+            raise MlflowException(
+                f"Run {run_id} is not older than the required age. "
+                f"Only runs older than {older_than} can be deleted."
```

Comment:
Can we add `error_code=INVALID_PARAMETER_VALUE` here?

Reply:
Added!